fuzzme.cpp

```diff
@@ -0,0 +1,9 @@
+#include "base64.h"
+#include <iostream>
```

Comment:
I would rename this file to indicate what is being fuzzed (`fuzzme` is not clear)